specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/ipTags.json

```diff
@@ -0,0 +1,406 @@
+      "get": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_List",
+        "description": "Gets all IpTags in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ipTag resources.",
+            "schema": {
+              "$ref": "#/definitions/IpTagsListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List_IpTags": {
+            "$ref": "./examples/IpTagsListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IpTagsListResult": {
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpTags"
+          },
+          "description": "The list of iptag information resources."
+        }
+      },
+      "description": "Response for the ListIpTags API service call."
+    },
+    "IpTags": {
+      "properties": {
+        "provisioningState": {
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the firewall policy resource."
+        },
+        "changeNumber": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The iteration number of the change."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IpTagsPropertiesFormat",
+          "description": "Properties of the IpTags."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "The IpTags resource information."
+    },
+    "IpTagsPropertiesFormat": {
+      "properties": {
+        "subgroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpTagsSubGroupPropertiesFormat"
+          },
+          "readOnly": true,
+          "description": "Properties of the iptag information."
+        },
+        "reference_resources": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of references to Azure resources that this IpTags is associated with",
+          "items": {
+            "$ref": "./network.json#/definitions/SubResource"
+          }
+        }
+      }
+    },
+    "IpTagsSubGroupPropertiesFormat": {
+      "properties": {
+        "name": {
```

Comment:
If `subgroups` is read-only in `IpTagsPropertiesFormat`, should all props of `IpTagsSubGroupPropertiesFormat` be read-only?

Reply:
Removed read-only from subgroups